autosklearn/metrics/__init__.py

```diff
@@ -341,19 +341,22 @@ def calculate_score(
     prediction: np.ndarray,
     task_type: int,
     metric: Scorer,
-    all_scoring_functions: bool = False
+    scoring_functions: list = None
```

Comment:
This should be something like `scoring_functions: List[Scorer] = None`, where `Scorer` is the scorer class in `autosklearn.metric`.

Reply:
Made necessary changes.